lib/drawers/publish-layout.vue

```diff
@@ -324,6 +326,7 @@
       },
       clearScheduleForm() {
         this.dateValue = null;
+        this.timeValue = null;
```

Comment:
This property needs to be cleared too in order to reflect the changes in the UI.